### DISCUSSION TITLE
Team `dura`

### DISCUSSION THREAD

**Discussion by tkellogg** [tkellogg/dura#81] [General]
A few community members have stepped up to lead specific areas of responsibility.

# Areas
Being the area lead means you’re responsible for moving us closer to the goal. That means closing PRs & issues, opening issues when help is needed, and writing code when necessary. In general, focus on completing the goal while upholding our values.

## Distribution
@drupol has had success integrating [dura into NixOS](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/development/tools/misc/dura/default.nix#L27). He will be taking responsibility for integrating `dura` into all package managers & operating systems.

## Stability
@JakeStanger has closed out the most pull requests around quality & code cleanliness. He will focus on all issues tagged "stability” and generally moving us toward a stable tool.

## Branch Management
@tkellogg will focus on branch management and generally reducing the number of branches a user has to wade through.

# Values
We value (in this order)
1. A useful & reliable tool — Users should see improvements, not regressions, from release-to-release
2. Thriving community — We believe that the long-term utility of `dura` increases as the community thrives
3. Quality contributions  — Contributors become involved to improve themselves just as much as to produce a gift to others. Maintaining a high bar for contributions is in line with everyone's interests.

We can adjust this charter over time as needs & constraints change.

**Comment by drupol** [comment]
Sounds good for me !

The idea is to improve my rust skills so, I'll be around :)

**Comment by JakeStanger** [comment]
That sounds like a plan, I'm all for a bit of structure. 

Some bits I think would help with enforcing that a bit and managing the project (I might be jumping the gun here a bit so no offense taken if you'd rather not yet):

- Setting up labels for issues/PRs based on the above areas would be great for managing our own work. These can be automated using actions, although I'm not sure how we'd do that or how much we'd get out of that.
- Having issues assigned to release milestones helps formalize releases.
- A project board might be a bit overkill at the moment, but they're great for bringing both of the above together to visualize where we're at.

**Reply by tkellogg** [reply]
I actually don't think a project board is overkill. The main reason I haven't done a lot of this is that I didn't want to commit other people's time. Having these areas assigned aims to address that, to some extent. 

Let's start with release milestones. That seems like a fairly big value add for us. Beyond that, I would love to see what automation we can apply, maybe @JakeStanger can lead the way there.

**Reply by JakeStanger** [reply]
I've plenty of experience with CI so more than happy to take that on :)
